recipes-core/uboot-env/files/update_ustate.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+#
+# Copyright (c) Siemens AG, 2021
+#
+# Authors:
+#  Quirin Gylstorff <quirin.gylstorff@siemens.com>
+#
+# SPDX-License-Identifier: MIT
+
+
+USTATE=$(fw_printenv -n ustate)
+
+if [ "${USTATE}" = "2" ]; then
+    fw_setenv ustate 0
+fi
```

Comment:
This file seems unrelated to the goal of patching the env for boot switching. Who is using it?

Reply:
That file is at the wrong location. Will move it to the correct location and add the missing parts.